scripts/import/uniprot/uniprot_importer.py

```diff
@@ -88,21 +90,40 @@ def fetch_all_data():
     print(f"Uniprot data successfully fetched via Uniprot API ({len(total_items)} entries)")
     return total_items
 
+
 # Function to insert Uniprot data to database
 def insert_uniprot_data(db_host, db_port, db_name, db_user, db_password, total_items):
```

Comment:
Suggesting that we break this into smaller functions

Reply:
I agree, this method can be improved but this PR is only a small update to be able to run it for the release.
Further improvements should be done later with more time.